### PR TITLE
cli: Add a message telling user to do 'quilt show' after 'quilt run'

### PR DESCRIPTION
**Pull request by aegamesi**
Previously, after a successful 'quilt run', there would be no user
feedback. This commit adds a hint telling the user to use 'quilt show'
to check the status of their deployment.

### CODE REVIEW

File: cli/command/run.go [hunk 1]
```diff
@@ -112,6 +112,8 @@ func (rCmd *Run) Run() int {
 		return 1
 	}
 
+	fmt.Println("Your blueprint is being deployed. " +
+		"You can check its status with `quilt show`.")
```
Comment (luise) [inline cli/command/run.go]: We're trying to use "you" a little less in docs and so, so how about this?
```
The blueprint is being deployed. Check its status with `quilt show`
```

Comment (luise) [reply cli/command/run.go]: Or maybe somewhere in between: 
```
Your blueprint is being deployed. Check its status with `quilt show`
```

Comment (kayousterhout) [reply cli/command/run.go]: Another idea that's a little more specific is `The Quilt daemon is deploying the blueprint. Check its status with ...`

Comment (luise) [reply cli/command/run.go]: I'm not sure the users need to worry about the Daemon here. It's more of a "how" than "what". But I don't feel strongly about it.

**Comment by aegamesi** [comment]
@luise I've made the changes you requested

**Pull request merged**